test/scripts/upgrade-ovn.sh

```diff
@@ -30,7 +30,10 @@ kubectl_wait_pods() {
 
 kubectl_wait_daemonset(){
   # takes one daemonset and makes sure its desiredNumberScheduled and numberReady are equal
+  local retries=0
+  local attempts=30
```

Comment:
looks like this will wait 15 minutes, is 15 min too long for a daemonset to rollout in kind?

Reply:
I reduced it by half, if it doesn't fail we should be good to go. I didn't really time it. There were errors though during my testing that sometimes polling started or exited too early. I am sure that's all fixed now. Squashed the commits.